eventing/README.md

```diff
@@ -43,10 +48,13 @@ This allows message delivery in a cluster to vary based on requirements, so that
 some events might be handled by an in-memory implementation while others would
 be persisted using Kafka or NATS Streaming.
 
+### Future design goals
+
 The focus for the next Eventing release will be to enable easy implementation of
 event sources. Sources manage registration and delivery of events from external
 systems using Kubernetes
 [Custom Resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
+Learn more about Eventing development in the [Events work group](https://github.com/knative/docs/blob/master/community/WORKING-GROUPS.md#events).
```

Comment:
s/Events/Eventing/

"Events" can be confused with attending something like Kubecon.

Reply:
I initially wrote "Eventing work group" but on that community page, its labelled "Events" work group (sounds like the WORKING-GROUPS.md page needs a update).